src/installer.archive.ts

```diff
@@ -395,38 +460,19 @@ export const installArchiveMod: VortexWrappedInstallFunc = (
 };
 
 //
-// Including Archives with other stuff
+// Internal API for including in other installers
 //
 
-const extraCanonArchiveLayoutsAllowedInOtherModTypes = [
-  archiveCanonWithXLLayout,
-  archiveCanonLayout,
-  archiveHeritageLayout,
-];
-
-export const detectExtraArchiveLayouts = (fileTree: FileTree): boolean =>
-  detectArchiveCanonWithXLLayout(fileTree) ||
-  detectArchiveCanonLayout(fileTree) ||
-  detectArchiveHeritageLayout(fileTree);
-
 export const extraCanonArchiveInstructions = (
   api: VortexApi,
   fileTree: FileTree,
 ): Instructions => {
-  const archiveInstructionsToUse = useFirstMatchingLayoutForInstructions(
-    api,
-    undefined,
-    fileTree,
-    extraCanonArchiveLayoutsAllowedInOtherModTypes,
-  );
+  const canonicalInstructions = instructionsForCanonicalExtras(api, fileTree);
 
-  if (
-    archiveInstructionsToUse === NoInstructions.NoMatch ||
-    archiveInstructionsToUse === InvalidLayout.Conflict
-  ) {
-    api.log(`debug`, `${InstallerType.Archive}: No valid extra canon archives`);
-    return { kind: NoLayout.Optional, instructions: [] };
-  }
+  /*
+  const transformedInstructions =
+    maybeTransformToREDmod(api, canonicalInstructions, fileTree);
+    */
```

Comment:
Debug cruft or something? Maybe remove it?
```suggestion
```

Reply:
Yeah, just upcoming work. I'll leave it since it's commented out, just to avoid any possible merge issues :D